### PR TITLE
[ticket/16284] Move 4.0 migrations to v400 folder

### PR DESCRIPTION
**Pull request by marc1706**
Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16284

### CODE REVIEW

File: phpBB/phpbb/db/migration/data/v400/add_storage_permission.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v400;
+
+use phpbb\db\migration\migration;
+
+class add_storage_permission extends migration
```
Comment (rubencm) [inline phpBB/phpbb/db/migration/data/v400/add_storage_permission.php]: The depends_on is missing (yes, i know i did this)

File: phpBB/phpbb/db/migration/data/v400/extensions_composer.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v400;
+
+use phpbb\db\migration\migration;
+
+class extensions_composer extends migration
```
Comment (rubencm) [inline phpBB/phpbb/db/migration/data/v400/extensions_composer.php]: The depends_on is missing

File: phpBB/phpbb/db/migration/data/v400/remove_attachment_download_mode.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v400;
+
+use phpbb\db\migration\migration;
+
+class remove_attachment_download_mode extends migration
```
Comment (rubencm) [inline phpBB/phpbb/db/migration/data/v400/remove_attachment_download_mode.php]: The depends_on is missing

File: phpBB/phpbb/db/migration/data/v400/storage_attachment.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v400;
+
+use phpbb\db\migration\migration;
+use phpbb\storage\provider\local;
+
+class storage_attachment extends migration
```
Comment (rubencm) [inline phpBB/phpbb/db/migration/data/v400/storage_attachment.php]: The depends_on is missing

**Comment by marc1706** [comment]
@rubencm should be good now

**Comment by rubencm** [comment]
Yes, I don't see anything else

**Pull request merged**